src/Array.mo

```diff
@@ -6,7 +6,9 @@ import { nyi = todo } "Debug";
 
 module {
   
-  public func init<T>(size : Nat, initValue : T) : [var T] = Prim.Array_init<T>(size, initValue);
+  public func new<T>() : [T] = [];
```

Comment:
I'm slightly worried about using new (here and elsewere, e.g lists) in case we want to use it for classes at  some point. 
Does empty() work  as well? That would aslo work nicely with isEmpty() for structures that provide it.

Reply:
That's a good point about leaving this open for classes in the future. I'll create another PR for this.